src/stores/ImageFittingStore.ts

```diff
@@ -127,11 +130,15 @@ export class ImageFittingStore {
         log += this.getRegionInfoLog(regionId, fovInfo) + "\n";
         log += fittingLog + "\n";
 
-        const toFixFormat = (param: string, value: number | string, error: number, unit: string): string => {
-            return `${param} = ${typeof value === "string" ? value : value?.toFixed(6)} +/- ${error?.toFixed(6)}${unit ? ` (${unit})` : ""}\n`;
+        const toFixFormat = (param: string, value: number | string, error: number, unit: string, fixed: boolean): string => {
+            const valueString = typeof value === "string" ? value : value?.toFixed(6);
+            const errorString = fixed ? "" : " +/- " + error?.toFixed(6);
```

Comment:
can we use the unicode [plus-minus](https://www.compart.com/en/unicode/U+00B1#:~:text=%E2%80%9C%C2%B1%E2%80%9D%20U%2B00B1%20Plus%2DMinus%20Sign%20Unicode%20Character) sign instead? `±`

Reply:
I am ok with this. 👍

Reply:
changed to unicode plus-minus sign for both image fitting and z profile fitting